.travis.yml

```diff
@@ -13,4 +13,6 @@ script:
   - julia -e 'Pkg.clone("https://github.com/mbauman/Tuples.jl"); Pkg.clone("https://github.com/mbauman/RangeArrays.jl")'
   - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("AxisArrays"); Pkg.test("AxisArrays"; coverage=true)'
 after_success:
-    - julia -e 'cd(Pkg.dir("AxisArrays")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'cd(Pkg.dir("AxisArrays")); Pkg.add("Coverage"); using Coverage; Coveralls.submit(Coveralls.process_folder())'
+  - julia -e 'Pkg.add("Documenter")'
+  - julia -e 'cd(Pkg.dir("AxisArrays")); include(joinpath("doc", "make.jl"))'
```

Comment:
👍  thanks!  I think I got all the keys set up now, too, so we'll see if it works after this push.  If you'd prefer, I can now move the folder back, but the Documenter setup tasks seemed to assume this layout… so this was the path of least resistance.